contrib/utilities/indent_common.sh

```diff
@@ -67,6 +67,29 @@ checks() {
 #
 export TMPDIR="${TMPDIR:-/tmp}"
 
+#
+# Depending on whether REPORT_ONLY is set to true, or false, either report
+# an issue or print a status
+#
+
+export REPORT_ONLY="${REPORT_ONLY:-false}"
+
+fix_or_report()
```

Comment:
can you document this macro and its arguments?

Reply:
This is now documented in `indent` and `indent-branch`. The use is mainly internal, so I did not touch the user documentation.

Reply:
Alternatively, we will have to look at the documentation a bit closer again once the dust settled anyway. (E.g., we haven't done the `indent-branch` -> `indent` switch yet.)

Reply:
Ah, you mean the function.

Reply:
Correct.